src/Models/Constants/TelemetryConstants.ts

```diff
@@ -188,26 +188,39 @@ export const TelemetryEvents = {
                 ParseDataHistory: {
                     eventName: 'ParseConfig.DataHistoryExplorerKpis',
                     properties: {
+                        modalId: 'modalId',
                         countSeries: 'countSeries'
                     }
                 }
             },
             UserAction: {
                 /** opening modal to explore data history */
                 OpenModal: {
-                    eventName: 'DataHistoryExplorer.AddSeries'
+                    eventName: 'DataHistoryExplorer.OpenModal',
+                    properties: {
+                        modalId: 'modalId'
```

Comment:
we want all these properties to always share the same property name, can we use a constant here across the events?

Reply:
do you mean using something like this? 
![image](https://user-images.githubusercontent.com/45217314/219458041-0db54d50-4be9-420e-b131-6e51904c3462.png)

Reply:
Okay I made a recent change as far as I understood from your comments, see if that is what you referring

Reply:
![image](https://user-images.githubusercontent.com/45217314/219460921-18f9dc33-ff36-4f2b-af21-cd3f0e942c8a.png)